.github/workflows/pipeline_swfs_test.yaml

```diff
@@ -0,0 +1,103 @@
+name: Deploy and test Kubeflow Pipelines manifests with seaweedfs and m2m auth in KinD
+on:
+  pull_request:
+    paths:
+    - .github/workflows/pipeline_swfs_test.yaml
+    - apps/pipeline/upstream/**
+    - tests/gh-actions/kind-cluster.yaml
+    - tests/gh-actions/install_kind.sh
+    - tests/gh-actions/install_kustomize.sh
+    - tests/gh-actions/install_istio.sh
+    - tests/gh-actions/install_cert_manager.sh
+    - common/cert-manager/**
+    - common/oidc-client/oauth2-proxy/**
+    - common/istio*/**
+    - tests/gh-actions/install_istio_with_ext_auth.sh
```

Comment:
This will probably fail since the file moved https://github.com/kubeflow/manifests/blob/master/tests/gh-actions/install_istio.sh and we need to install oauth2-proxy now separately as changed in the large authentication commit yesterday.